### PR TITLE
chore: adap REUSE license

### PR DESCRIPTION
**Pull request by tingwei1123**
Distinguish between self-developed code and non-self-developed code
Log: Distinguish between self-developed code and non-self-developed code

### CODE REVIEW

File: src/grand-search/CMakeLists.txt [hunk 1]
```diff
@@ -26,6 +26,15 @@ find_package(Qt5 COMPONENTS
     DBus
 REQUIRED)
 
+set(3RDPARTY
```
Comment (Clauszy) [inline src/grand-search/CMakeLists.txt:29]: DBUSSRC

File: src/grand-search/CMakeLists.txt [hunk 2]
```diff
@@ -26,6 +26,15 @@ find_package(Qt5 COMPONENTS
...
+    # dbus接口
+    ${CMAKE_SOURCE_DIR}/3rdparty/interfaces/daemongrandsearchinterface.h
```
Comment (Clauszy) [inline src/grand-search/CMakeLists.txt:31]: 在根cmake中定义${CMAKE_SOURCE_DIR}/3rdparty/， 这里太长了

File: src/grand-search/main.cpp [hunk 3]
```diff
@@ -10,7 +10,7 @@
 #include "business/query/querycontroller.h"
 #include "business/matchresult/matchcontroller.h"
 #include "contacts/services/grandsearchservice.h"
-#include "contacts/services/grandsearchserviceadaptor.h"
+#include "services/grandsearchserviceadaptor.h"
```
Comment (Clauszy) [inline src/grand-search/main.cpp:13]: 分块，下移

**Review by Clauszy — CHANGES_REQUESTED** [review]

**Review by Clauszy — APPROVED** [review]

**Comment by tingwei1123** [comment]
-merge

**Pull request merged**